src/components/__tests__/Button.spec.tsx

```diff
@@ -11,81 +11,79 @@ import resetAllMocks = jest.resetAllMocks
 
 type ButtonProps = ComponentProps<typeof Button>
 
-describe('Components', () => {
-  describe('Button', () => {
-    const onPressMock = jest.fn()
+describe('Button', () => {
```

Comment:
Only removed the wrapping in `describe('Components', () => {...})`, same for other tests (except from `DisciplineItem.spec.tsx`).